--- a/src/command_network.cc
+++ b/src/command_network.cc
@@ -250,6 +250,26 @@
   CMD2_VAR_BOOL    ("protocol.pex",            true);
   CMD2_ANY_LIST    ("protocol.encryption.set", std::bind(&apply_encryption, std::placeholders::_2));

+  CMD2_ANY         ("colors.color_inactive_fg",       std::bind(&ui::Root::get_color_inactive_fg,   control->ui()));
+  CMD2_ANY_VALUE_V ("colors.color_inactive_fg.set",   std::bind(&ui::Root::set_color_inactive_fg,   control->ui(), std::placeholders::_2));
+  CMD2_ANY         ("colors.color_inactive_bg",       std::bind(&ui::Root::get_color_inactive_bg,   control->ui()));
+  CMD2_ANY_VALUE_V ("colors.color_inactive_bg.set",   std::bind(&ui::Root::set_color_inactive_bg,   control->ui(), std::placeholders::_2));
+
+  CMD2_ANY         ("colors.color_dead_fg",       std::bind(&ui::Root::get_color_dead_fg,   control->ui()));
+  CMD2_ANY_VALUE_V ("colors.color_dead_fg.set",   std::bind(&ui::Root::set_color_dead_fg,   control->ui(), std::placeholders::_2));
+  CMD2_ANY         ("colors.color_dead_bg",       std::bind(&ui::Root::get_color_dead_bg,   control->ui()));
+  CMD2_ANY_VALUE_V ("colors.color_dead_bg.set",   std::bind(&ui::Root::set_color_dead_bg,   control->ui(), std::placeholders::_2));
+
+  CMD2_ANY         ("colors.color_active_fg",       std::bind(&ui::Root::get_color_active_fg,   control->ui()));
+  CMD2_ANY_VALUE_V ("colors.color_active_fg.set",   std::bind(&ui::Root::set_color_active_fg,   control->ui(), std::placeholders::_2));
+  CMD2_ANY         ("colors.color_active_bg",       std::bind(&ui::Root::get_color_active_bg,   control->ui()));
+  CMD2_ANY_VALUE_V ("colors.color_active_bg.set",   std::bind(&ui::Root::set_color_active_bg,   control->ui(), std::placeholders::_2));
+
+  CMD2_ANY         ("colors.color_finished_fg",       std::bind(&ui::Root::get_color_finished_fg,   control->ui()));
+  CMD2_ANY_VALUE_V ("colors.color_finished_fg.set",   std::bind(&ui::Root::set_color_finished_fg,   control->ui(), std::placeholders::_2));
+  CMD2_ANY         ("colors.color_finished_bg",       std::bind(&ui::Root::get_color_finished_bg,   control->ui()));
+  CMD2_ANY_VALUE_V ("colors.color_finished_bg.set",   std::bind(&ui::Root::set_color_finished_bg,   control->ui(), std::placeholders::_2));
+
   CMD2_VAR_STRING  ("protocol.connection.leech", "leech");
   CMD2_VAR_STRING  ("protocol.connection.seed",  "seed");